pulp_2to3_migration/app/plugin/rpm/xml_utils.py

```diff
@@ -229,3 +230,20 @@ def newpkgcb(pkgId, name, arch):
     cr.xml_parse_filelists_snippet(filelists_xml, newpkgcb=newpkgcb)
     cr.xml_parse_other_snippet(other_xml, newpkgcb=newpkgcb)
     return list(packages.values())[0]
+
+
+def _decompress_repodata(compressed_repodata):
```

Comment:
Is there a reason to have it with underscore at the beginning?
Conventionally, it's used for non-public methods/functions, however this one is already used in a different module.
Thoughts?

Reply:
no reason, i will change that. i probably forgot when i was moving the code around.